examples/pytorch/question-answering/run_qa_no_trainer.py

```diff
@@ -292,8 +303,10 @@ def main():
             data_files["train"] = args.train_file
         if args.validation_file is not None:
             data_files["validation"] = args.validation_file
+        if args.test_file is not None:
+            data_files["test"] = args.test_file
         extension = args.train_file.split(".")[-1]
-        raw_datasets = load_dataset(extension, data_files=data_files)
+        raw_datasets = load_dataset(extension, data_files=data_files, field="data")
```

Comment:
why are we adding a new arg here? apologies if I missed the discussion about it.

Reply:
Actually, This part was not discussed in that PR but i added this arg because the input JSON file is having below format,
```json
{
    "version": 2.0,
    "data": [
        {
            "id": "56ddde6b9a695914005b9628",
            "question": "In what country is Normandy located?",
            "context": "The Normans (Norman: Nourmands; ...
}
```
so if we don't add these args it will not parse data properly.

In other examples like `run_qa.py`
https://github.com/huggingface/transformers/blob/ad1f7bef13f03287af00f819605d696138a5e6ec/examples/pytorch/question-answering/run_qa.py#L272
and 
https://github.com/huggingface/transformers/blob/ad1f7bef13f03287af00f819605d696138a5e6ec/examples/pytorch/question-answering/run_qa_beam_search.py#L270

so I thought it should be necessary. 

If we don't do these changes while using local files the script was giving an error.

Reply:
Thank you for explaining, why it was added, @bhadreshpsavani